.gitignore

```diff
@@ -19,4 +19,4 @@ report.json
 
 # Generated binaries
 /_examples/docscan/docscan
-
+/_examples/idv
```

Comment:
i think there's an `_examples` specific `.gitignore` btw https://github.com/getyoti/yoti-go-sdk/blob/SDK-2230-Expose-share-v2-api/_examples/.gitignore

Reply:
and we can also add the `.exe` variants of all these binaries too, and remove them from the repo

Reply:
updated